### PR TITLE
Add documentation on how to convert QuartzNet model

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Add documentation on how to convert QuartzNet model*

### Tickets:
 - *38854*

### CODE REVIEW

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Convert PyTorch* QuartzNet to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_QuartzNet}
+
+[NeMo project](https://github.com/NVIDIA/NeMo) provides QuartzNet model.
+
+## Download the Pre-Trained QuartzNet Model
+
+To download pre-trained model please refer to [NeMo Developer Guide](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/).
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:7]: I moved by this link and didn't find any mention about this model. Can you make this link more concrete please?

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Here are the instructions that can be used to obtain QuartzNet in ONNX* format.
+```python
+import nemo
+import nemo.collections.asr as nemo_asr
+
+quartznet = nemo_asr.models.ASRConvCTCModel.from_pretrained(model_info='QuartzNet15x5-En')
+# Export QuartzNet model to ONNX* format
+quartznet.exporrt('qn.onnx')
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:15]: misprint: exporrt

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+```
+This code will produce 3 ONNX* model files: `encoder_qt.onnx`, `decoder_qt.onnx`, `qn.onnx`.
+They are `decoder`, `encoder` and a combined `decoder(encoder(x))` models respectively.
+
+## Convert ONNX* QuartzNet model to IR
+
+```sh
+./mo.py --input_model encoder_qt.onnx --input_shape [B,64,X]
+./mo.py --input_model decoder_qt.onnx --input_shape [B,1024,Y]
+```
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:25]: I think it is better to elaborate what each dimension means and why one of them is hardcoded.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+quartznet = nemo_asr.models.ASRConvCTCModel.from_pretrained(model_info='QuartzNet15x5-En')
+# Export QuartzNet model to ONNX* format
+quartznet.export('qn.onnx')
+```
+This code will produce 3 ONNX* model files: `encoder_qt.onnx`, `decoder_qt.onnx`, `qn.onnx`.
+They are `decoder`, `encoder` and a combined `decoder(encoder(x))` models respectively.
+
+## Convert ONNX* QuartzNet model to IR
+
+```sh
```
Comment (sadolini) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:22]: Not clear why we convert only decoder and encoder separately and don't convert combination (if it possible or not, needed or useless).

**Review by rkazants — APPROVED** [review]

**Review by yekruglov — APPROVED** [review]

**Review by sadolini — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Comment by lazarevevgeny** [comment]
@andrew-zaytsev , @ntyukaev , please, review

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Convert PyTorch* QuartzNet to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_QuartzNet}
+
+[NeMo project](https://github.com/NVIDIA/NeMo) provides QuartzNet model.
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:3]:
```suggestion
[NeMo project](https://github.com/NVIDIA/NeMo) provides the QuartzNet model.
```

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Download the Pre-Trained QuartzNet Model
+
+To download pre-trained model please refer to [NeMo Speech Models Catalog](https://ngc.nvidia.com/catalog/models/nvidia:nemospeechmodels).
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:7]:
```suggestion
To download the pre-trained model, refer to the [NeMo Speech Models Catalog](https://ngc.nvidia.com/catalog/models/nvidia:nemospeechmodels).
```

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+Here are the instructions that can be used to obtain QuartzNet in ONNX* format.
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:8]:
```suggestion
Here are the instructions on how to obtain QuartzNet in ONNX* format.
```

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+```python
+import nemo
+import nemo.collections.asr as nemo_asr
+
+quartznet = nemo_asr.models.ASRConvCTCModel.from_pretrained(model_info='QuartzNet15x5-En')
+# Export QuartzNet model to ONNX* format
+quartznet.export('qn.onnx')
+```
+This code will produce 3 ONNX* model files: `encoder_qt.onnx`, `decoder_qt.onnx`, `qn.onnx`.
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:17]:
```suggestion
This code produces 3 ONNX* model files: `encoder_qt.onnx`, `decoder_qt.onnx`, `qn.onnx`.
```

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+They are `decoder`, `encoder` and a combined `decoder(encoder(x))` models respectively.
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:18]:
```suggestion
They are `decoder`, `encoder` and a combined `decoder(encoder(x))` models, respectively.
```

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+## Convert ONNX* QuartzNet model to IR
+
+If using combined model:
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md:22]:
```suggestion
If using a combined model:
```

**Review by avladimi — APPROVED** [review]

**Pull request merged**